events/start_handler.go

```diff
@@ -64,21 +69,25 @@ func (h *StartHandler) Handle(event *docker.APIEvents) error {
 	return nil
 }
 
-func (h *StartHandler) getRancherIP(c *docker.Container) string {
+func (h *StartHandler) getRancherIP(c *docker.Container) (string, error) {
 	for _, env := range c.Config.Env {
 		if strings.HasPrefix(env, RancherIPKey) {
-			return strings.TrimPrefix(env, RancherIPKey)
+			return strings.TrimPrefix(env, RancherIPKey), nil
 		}
 	}
 
 	filePath := path.Join(getContainerStateDir(), c.ID)
 	if _, err := os.Stat(filePath); err == nil {
+		time.Sleep(time.Millisecond * 200)
```

Comment:
I'm listening for file `create` events in the fsnotify watcher. There must be a race condition where the file is not completely written, because I'll inconsistently get a "unexpected end of json" error. But with this sleep, I no longer get that error. I intend this to be a temporary fix, since we want to move away from fsnotify all together.